### DISCUSSION TITLE
Install guide from Docker Hub images

### DISCUSSION THREAD

**Discussion by ewallz** [ludeknovy/jtl-reporter#251] [Q&A] [answered]
Hi, i saw the latest image on docker hub, the 'latest' one support arm64 for both FE/BE. Does it supports ARM now?
And is there guide on steps to install from docker hub images?

**Comment by ludeknovy** [comment]
Hi @ewallz! The arm images were introduced very recently. I havent tested it myself, but if you run the docker compose on the arm architecture, it should download it automatically, or enforce it by specifying the platform - https://docs.docker.com/compose/compose-file/compose-file-v2/#platform

**Reply by ewallz** [reply]
I edited the docker-compose file & successfully installed it on my arm server using the 'latest' tag images. All the services is running fine now. Importing csv working fine. Just some error toasts during settings, but could not remember what sort of notice.
Anyway everything seemed just working great.

**Reply by ludeknovy** [reply, marked as answer]
Good! The error you have encountered will be most likely a bug but not related to arm

**Reply by ludeknovy** [reply]
If you manage to reproduce it, please report it and I will fix it. Thanks
